helm/mds/templates/egress.yaml

```diff
@@ -6,7 +7,7 @@ metadata:
   namespace: {{ .Release.Namespace }}
 spec:
   hosts:
-  - {{ printf "nats.%s.svc.cluster.local" $natsNamespace }}
+  - {{ printf "%s-nats-server.%s.svc.cluster.local" $natsNamespace $natsNamespace }}
```

Comment:
yeah we could fix/reduce the redundancy.

this might appear in other templates as well so we should be consistent regardless.

likely just picked the pattern up and continued using it.

Reply:
I think this should actually just pick up the nats endpoint name from `.Values.natsUrl` or something like that.  We shouldn't be trying to construct what the endpoint is from a different software install.  What if we're using an external (hosted) nats server for example?  I'd recommend to replace `.Values.natsNamespace` with `.Values.natsUrl`

Reply:
i'd like to push this to a subsequent incremental patch given as implemented it follows the current conventions for cluster-hosted endpoint resolution